_posts/2025-01-15-moving-data-between-JBOD-disks-using-cruise-control.md

```diff
@@ -0,0 +1,546 @@
+---
+layout: post
+title: "Moving data between JBOD disks using Cruise Control"
+date: 2025-01-15
+author: shubham_rawat
+---
+
+Apache Kafka is a platform which provides durability and fault tolerance by storing messages on disks and JBOD storage is one of the storage configuration types supported by Strimzi.
+The JBOD data storage configuration allows Kafka brokers to make use of multiple disks.
```

Comment:
There seems to be a bit of a disconnect / big jump in this sentence. Maybe you can lead it out with somehting like this?

```suggestion
Apache Kafka is a platform that provides durability and fault tolerance by storing messages on persistent volumes.
In most cases, each Kafka broker will use one persistent volume.
However, it is also possible to use multiple volumes for each broker.
This configuration is called JBOD storage.
```